net/ovn/patches/0004-fix-use-of-possible-uninitialized-var.patch

```diff
@@ -0,0 +1,21 @@
+Fix the following compilation errors
+
+ic/ovn-ic.c: In function 'main':
+ic/ovn-ic.c:689:5: warning: 'ls' may be used uninitialized in this function [-Wmaybe-uninitialized]
+  689 |     nbrec_logical_switch_update_ports_addvalue(ls, lsp);
+      |     ^
+ic/ovn-ic.c:430:40: note: 'ls' was declared here
+  430 |     const struct nbrec_logical_switch *ls;
+      |                                        ^
```

Comment:
This patch was submitted upstream for review: https://github.com/ovn-org/ovn/pull/121